Source/WebKit/UIProcess/Cocoa/NavigationState.h

```diff
@@ -283,6 +286,7 @@ class NavigationState final : public PageLoadState::Observer {
 #if HAVE(APP_SSO)
         bool webViewDecidePolicyForSOAuthorizationLoadWithCurrentPolicyForExtensionCompletionHandler : 1;
 #endif
+        bool didGeneratePageLoadTiming : 1;
```

Comment:
I think this should start with `webView`, like the other ones, since the new method is `webView: something..`

Reply:
Sure, will fix.